lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -20,12 +20,11 @@ import { MatCheckboxChange } from '@angular/material';
 import { SearchService, TranslationService } from '@alfresco/adf-core';
 import { SearchQueryBuilderService } from '../../search-query-builder.service';
 import { FacetFieldBucket } from '../../facet-field-bucket.interface';
-import { ResponseFacetQueryList } from './models/response-facet-query-list.model';
-import { FacetQuery } from '../../facet-query.interface';
 import { FacetField } from '../../facet-field.interface';
 import { SearchFilterList } from './models/search-filter-list.model';
 import { takeWhile } from 'rxjs/operators';
 import { ResultSetPaging } from '@alfresco/js-api';
+import { GenericBucket, GenericFacetResponse, ResultSetContext } from 'alfresco-js-api-node';
```

Comment:
you can not import from alfresco-js-api-node, this is used only in the e2e test. Please import it from the @alfresc/js-api